Firebase/CoreDiagnostics/FIRCDLibrary/FIRCoreDiagnostics.m

```diff
@@ -90,7 +90,7 @@
 /**
  * The file name to keep the unique install string.
  */
-static NSString *const kUniqueInstallFileName = @"FIREBASE_UNIQUE_INSTALL";
```

Comment:
Why removing the static's?  

If necessary, they should be better Firebase namespaced. Otherwise we'll break any of our users with the global variable `kUniqueInstallFileName`.

Reply:
statics were put back.